tensorboard/webapp/runs/views/runs_table/runs_table_test.ts

```diff
@@ -1530,10 +1530,10 @@ describe('runs_table', () => {
         ['LotR', 'The Silmarillion'],
       ]);
 
-      // Alias for Harry Potter contains "z". Since tf-run-selector matches
-      // regex against '<alias>/<run name>' instead of '<experiment>/<run name>'
-      // Harry Potter should match below.
-      // This regex matches:
+      // Alias for Harry Potter contains "z". Since legacy Polymer based
+      // tf-run-selector matches, and the new Angular run selector, regex
```

Comment:
What I am trying to say is that this behavior comes from the old Polymer-based tf-run-selector that is being adopted in the new Angular run selector. Would it be better to drop, "and the new Angular..." or perhaps put that phrase inside a parenthesis?

Reply:
Ah, maybe you just need to shift that fragment one word to the left and change your conjugation of your first use of "to match".

Since legacy Polymer-based tf-run-selector and the new Angular run selector match regex against `<alias>/<run_name>` instead of `<experiment>/<run name>`, the below regex matches: